### DISCUSSION TITLE
Is it possible to specify a path in experiments of type "pod-http-status"?

### DISCUSSION THREAD

**Discussion by Alan-Vlc** [litmuschaos/litmus#4423] [General]
Good morning,

About "Pod HTTP Status Code" type experiments I have the following doubt:

In my kubernetes cluster I have a pod that exposes a large number of endpoints and I would like to replace the response code of the http request to only one of them.
I can't find a way to do it (In chaos-mesh it is possible to determine the path on which we want to apply the experiment. Do you know how it would be possible?)

Thank you

**Comment by neelanjan00** [comment]
Hi @Alan-Vlc! 
Path filtering is not yet supported for HTTP chaos faults since they use TCP proxying which only works on the L3 layer.

**Reply by Alan-Vlc** [reply]
Good afternoon @neelanjan00 ,

Thank you very much for your answer.

Is there any expectation that this improvement will be implemented in the future?
In tools like chaos-mesh, path filtering is possible in this type of experiments. :)

**Reply by neelanjan00** [reply]
We might add it in the future but currently, it is not being developed. That being said, feel free to create an issue for this, we will be glad to help anyone who can contribute to adding this fault. 

CC: @ksatchit @ispeakc0de what do you think?

**Comment by Alan-Vlc** [comment]
Thanks for your clarification @neelanjan00 

I am going to open an issue so that it can be taken into account in the future. I think this improvement will add a lot of value to the experiments.

Thank you

**Comment by Alan-Vlc** [comment]
I am going to open an issue so that it can be taken into account in the future.
